Sources/RayTracer.Library/Serialization/Serializers/IntersectableListSerializer.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Text.Json;
+using RayTracer.Library.Extensions;
+using RayTracer.Library.Shapes;
+
+namespace RayTracer.Library.Serialization.Serializers;
+
+public class IntersectableListSerializer : SerializerBase<IntersectableListSerializer, IntersectableList>
+{
+    private readonly ISerializer<IIntersectable> _itemSerializer;
+
+    public IntersectableListSerializer()
+    {
+        _itemSerializer = new PolymorphicSerializer<IIntersectable>();
```

Comment:
`PolymorphicSerializer` is a singleton, you can use it directly via
`PolymorphicSerializer<IIntersectable>.Instance`